Dockerfile

```
@@ -1,29 +1,29 @@
 # This dockerfile is specific to building Multus for OpenShift
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.21 AS rhel9
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.25-openshift-4.22 AS rhel9
 ADD . /usr/src/plugins
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)